### PR TITLE
Feature/jkw timer ui

### LINKED ISSUE #38: Timer Component- Update UI & Refine Context
### Feature Categories:
- React
- Styling/CSS

**Describe the feature/Bug your gonna work on**
- We have the core of our timer setup, but need to write a little more code to refine the aesthetic/functional pieces
     - Aesthetic: there's a little bit of code in there to have our salmon circle tick down with the numbers
          - How can we access `timeLeft` from the `Timer` component given by `react-compound-timer`
     - Functional: our `Timer` is connected to the `SettingsContext`, but when a user changes their time in settings it's not updating on screen
          - After some troubleshooting yesterday, we determined that the update is happening in the `CountdownTimer` component, but isn't getting passed down to `Timer`'s state... 
          - Note from 10/26/2020: after implementing `Router`, this functional issue is resolved. The `Timer` component is now only rendered after settings have been updated. -jkw

### PR DESCRIPTION
**Pull request by jkwest-93**
# Som Timer Pull Request Form
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- After an initial take on the Timer, we needed to finish a few UI pieces so that the timer will count down. This PR adds some methods to tidy up this functionality
    - This PR also includes some additions to our colors variables, which are used in their hex code form in the `CountdownTimer` component.
- Removes some extra spaces from tabs across a number of components
- Closes #38 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
- Run `npm test` in the terminal to see all passing tests; run `npm start` to see this code in action.
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

- Here's a gif of this functionality! For some reason, GIPHY CAPTURE won't show the first color we're using (which is our `$salmon` variable...
countdown_withColor

### CODE REVIEW

File: src/Components/Timer/Timer.js [hunk 1]
```diff
@@ -4,87 +4,87 @@ import style from './Timer.module.scss';
...
+        startImmediately={false}
+        timeToUpdate={10}
+        checkpoints={[
+          {
+            time: 0,
+            callback: () => timerDone(),
+          }, 
+          /* These colors can be found in our variables.scss file (salmon progression) */
+          {
+            time: settings.workInterval * 45000,
```
Comment (rwilliams659) [inline src/Components/Timer/Timer.js:35]: Love this color changing functionality, so slick!

**Review by rwilliams659 — APPROVED** [review]
Glad we worked on some of this together, definitely deepened my understanding of the timer logic

File: src/Components/Timer/Timer.js (hunk 1 above)
Comment (jkwest-93) [reply src/Components/Timer/Timer.js:35]: Yes! Great idea @Abdeboskey to toggle these on quarter intervals.

@rwilliams659 I'm glad we could all take a look at it together as well! Naturally, I forgot to co-author these commits...

File: src/Components/Settings/Settings.js [hunk 2]
```diff
@@ -3,49 +3,49 @@ import style from './Settings.module.scss'
...
+          data-testid='breakInterval'
+        >
+          <option value={5}>5</option>
+          <option value={7}>7</option>
+          <option value={10}>10</option>
+        </select>
+        <p>minutes</p>
+      </div>
+    </div>
+  )
```
Comment (Abdeboskey) [inline src/Components/Settings/Settings.js:48]: Thanks for making these indentation adjustments Jake 🙏🏼

File: src/Components/Timer/Timer.js (hunk 1 above)
Comment (Abdeboskey) [reply src/Components/Timer/Timer.js:35]: Haha no worries on the co commits. I'm stoked that you were able to get this working, thanks for being down to spend time on it together!

**Review by Abdeboskey — APPROVED** [review]
Great work Jake! Fun to work with y'all on this as usual.

File: src/Components/Settings/Settings.js [hunk 3]
```diff
@@ -3,49 +3,49 @@ import style from './Settings.module.scss'
...
+    <div className={style.settingsContainer}>
+      <h2 className={style.settingsHeader}>Settings</h2>
+      <div className={style.intervalContainer}>
+        <p className={style.intervalLabel}>Select your work interval:</p>
+        <input
+          type='number'
+          min={0}
+          max={240}
+          id='workInterval'
+          placeholder={settings.workInterval}
```
Comment (sienna-kopf) [inline src/Components/Settings/Settings.js:25]: having this as a place holder is great!

File: src/Components/Settings/Settings.test.js [hunk 4]
```diff
@@ -4,48 +4,48 @@ import { fireEvent, render } from '@testing-library/react';
...
+      </SettingsProvider>
+    )
+
+    const workIntervalInput = getByTestId('workInterval')
+    const breakIntervalInput = getByTestId('breakInterval')
+
+    fireEvent.change(workIntervalInput, { target: { value: '45' } })
+    fireEvent.change(breakIntervalInput, { target: { value: '10' } })
+
+    expect(workIntervalInput.value).toBe('45')
```
Comment (sienna-kopf) [inline src/Components/Settings/Settings.test.js:48]: Yes testing!!!!

File: src/Components/Timer/Timer.js [hunk 5]
```diff
@@ -4,87 +4,87 @@ import style from './Timer.module.scss';
...
-    //         document.getElementById('base-timer-path-remaining').setAttribute('stroke-dasharray', circleDashArray)
-    // }
+  const timerDone = () => {
+    setView('content-selection')
+  }
 
-    const timerDone = () => {
-        setView('content-selection')
-    }
+  const countdownColorChange = (color) => {
```
Comment (sienna-kopf) [inline src/Components/Timer/Timer.js:15]: Thanks for including the screencast in the PR. I love how the circle is formed as the interval counts down!

**Review by sienna-kopf — APPROVED** [review]

**Pull request merged**